PluralKit.Bot/Services/EmbedService.cs

```diff
@@ -262,14 +262,14 @@ public async Task<Embed> CreateGroupEmbed(Context ctx, PKSystem system, PKGroup
 
         var eb = new EmbedBuilder()
             .Author(new Embed.EmbedAuthor(nameField, IconUrl: target.IconFor(pctx)))
-            .Color(color)
-            .Footer(new Embed.EmbedFooter(
-                $"System ID: {system.Hid} | Group ID: {target.Hid} | Created on {target.Created.FormatZoned(ctx.Zone)}"));
+            .Color(color);
+        
+        eb.Footer(new Embed.EmbedFooter($"System ID: {system.Hid} | Group ID: {target.Hid}{(target.MetadataPrivacy.CanAccess(pctx) ? $" | Created on {target.Created.FormatZoned(ctx.Zone)}" : "")}"));
 
         if (target.DescriptionPrivacy.CanAccess(ctx.LookupContextFor(target.System)))
             eb.Image(new Embed.EmbedImage(target.BannerImage));
 
-        if (target.DisplayName != null)
+        if (target.NamePrivacy.Get(pctx, target.DisplayName, null) != null)
```

Comment:
please just check directly against NamePrivacy instead of seeing if .Get returns null